tests/test_analyzer.py

```
@@ -1076,6 +1076,176 @@ async def fake_to_thread(func, *args, **kwargs):
+        # The key assertion: get_build_console was called, proving it went past
+        # the SUCCESS early-return guard.
+        try:
+            await analyze_job(
+                body,
+                merged,
+                ai_provider="claude",
+                ai_model="test-model",
+                job_id=None,
+            )
+        except Exception:
+            pass  # AI CLI sanity check failure is expected
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)